src/pytorch_lightning/callbacks/quantization.py

```diff
@@ -26,8 +26,8 @@
 from torch.quantization import FakeQuantizeBase
 
 import pytorch_lightning as pl
+from lightning_lite.utilities.imports import _TORCH_GREATER_EQUAL_1_11, _TORCH_GREATER_EQUAL_1_12
```

Comment:
can we do this in separate PR?

Reply:
It's just removing one duplicated variable

Reply:
hmm but your removing is 95% of this PR :P

Reply:
I would just update the PR name, but @carmocca I know may complain :)